data-prepper-plugins/http-source/README.md

```diff
@@ -60,9 +60,14 @@ create a plugin which implements [`ArmeriaHttpAuthenticationProvider`](../armeri
 
 ### SSL
 
-* ssl(Optional) => A `boolean` enables TLS/SSL. Default is ```false```.
-* ssl_certificate_file(Optional) => A `String` represents the SSL certificate chain file path. Required if ```ssl``` is set to ```true```.
-* ssl_key_file(Optional) => A `String` represents the SSL key file path. Only decrypted key file is supported. Required if ```ssl``` is set to ```true```.
+* ssl(Optional) => A `boolean` that enables TLS/SSL. Default is ```false```.
+* ssl_certificate_file(Optional) => A `String` that represents the SSL certificate chain file path or AWS S3 path. S3 path example `s3://<bucketName>/<path>`. Required if `ssl` is set to `true`.
```

Comment:
Relaying a message from another PR: Is `ssl_certificate_file` required only when `ssl` is true _and_ `use_acm_certificate_for_ssl` is false?

Reply:
With `ssl` enabled by default, they are no longer required as `ssl_certificate_file` and `ssl_key_file` have default values.

Reply:
Yes, we should clarify that here in the documentation.